ios/default.nix

```diff
@@ -278,7 +278,74 @@ nixpkgs.runCommand "${executableName}-app" (rec {
     # focus????
     focus/reflex-platform/scripts/run-in-ios-sim "$tmpdir/${executableName}.app"
   '';
-}) ''
+  portableDeployScript = nixpkgs.writeText "portable-deploy" ''
+    #!/usr/bin/env bash
+    set -eo pipefail
+
+    if (( "$#" < 1 )); then
+      echo "Usage: $0 [TEAM_ID]" >&2
+      exit 1
+    fi
+
+    TEAM_ID=$1
+    shift
+
+    set -euo pipefail
+
+    function cleanup {
+      if [ -n "$tmpdir" -a -d "$tmpdir" ]; then
+        echo "Cleaning up tmpdir" >&2
+        chmod -R +w $tmpdir
+        rm -fR $tmpdir
+      fi
+    }
+
+    trap cleanup EXIT
+
+    tmpdir=$(mktemp -d)
+    # Find the signer given the OU
+    signer=$({ security find-certificate -c 'iPhone Developer' -a; security find-certificate -c 'Apple Development' -a; } \
+      | grep '^    "alis"<blob>="' \
+      | sed 's|    "alis"<blob>="\(.*\)"$|\1|' \
+      | while read c; do \
+          security find-certificate -c "$c" -p \
+            | ${nixpkgs.libressl}/bin/openssl x509 -subject -noout; \
```

Comment:
When you say "without nix"...what do you mean?

Reply:
So portable-deploy needs nix to sign the binary, but once it's signed, we can just run the `./deploy` (found below) in the built tarball.

Reply:
Ah I see. Thanks.